.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+#### What does this PR do? \*
```

Comment:
Being picky, but this is the old PR template

Reply:
I think the react-app-template is outdated